TwilioVerifySDKTests/TwilioVerify/Sources/Models/Mocks/FakeFactorPayload.swift

```diff
@@ -44,6 +44,7 @@ struct FakeFactor: Factor {
   var identity: String
   var type: FactorType
   var createdAt: Date
+  var metadata: [String : String]?
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/8493b62acdbfcf89b7484fe450452cd8fa9157ed/TwilioVerifySDKTests/TwilioVerify/Sources/Models/Mocks/FakeFactorPayload.swift#L47"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>FakeFactorPayload.swift:47</code></td>
    </tr>
  </tbody>
</table>